src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingRuleDialog.xaml

```diff
@@ -0,0 +1,141 @@
+             xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+             xmlns:imagecatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:s="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
+             mc:Ignorable="d" 
+             d:DesignHeight="240" d:DesignWidth="480"
+             Height="240" Width="480"
+             MinHeight="240" MinWidth="480"
+             Title="{Binding ElementName=dialog, Path=DialogTitle}"
+             HasHelpButton="False"
+             FocusManager.FocusedElement="{Binding ElementName=NamingRuleTitle}"
+             ResizeMode="CanResizeWithGrip"
+             ShowInTaskbar="False"
+             HasDialogFrame="True"
+             WindowStartupLocation="CenterOwner">
+    <Window.Resources>
+        <Style TargetType="ListBoxItem">
+            <Setter Property="IsTabStop" 
+                    Value="False" />
+        </Style>
+        <Thickness x:Key="labelPadding">0, 5, 0, 2</Thickness>
+        <Thickness x:Key="okCancelButtonPadding">9,2,9,2</Thickness>
+        <Thickness x:Key="selectDeselectButtonPadding">9,2,9,2</Thickness>
+        <Thickness x:Key="textboxPadding">2</Thickness>
+    </Window.Resources>
+    <Grid Margin="6,6,11,11">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" MinWidth="50"/>
+            <ColumnDefinition MinWidth="250" Width="*"/>
+            <ColumnDefinition Width="Auto"/>
+            <ColumnDefinition Width="Auto"/>
+        </Grid.ColumnDefinitions>
+
+        <!-- Row 0 -->
+        <Label Grid.Row="0" Grid.Column="0" Content="{Binding ElementName=dialog, Path=NameLabelText}"/>
+        <TextBox x:Name="NamingRuleTitle" Text="{Binding Title, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" Grid.Row="0" Grid.Column="1" Grid.ColumnSpan="3" VerticalContentAlignment="Center"/>
+
+        <!-- Row 1 -->
+        <Label Grid.Row="1" Grid.Column="0" Margin="0 5 0 0" Content="{Binding ElementName=dialog, Path=SymbolSpecificationLabelText}"/>
+        <ComboBox ItemsSource="{Binding Path=SymbolSpecificationList}"
+                DisplayMemberPath="SymbolSpecName"
+                VerticalContentAlignment="Center"
+                SelectedIndex="{Binding Path=SelectedSymbolSpecificationIndex}" 
```

Comment:
looks like I may be off here, I see further `selectedStuff` that are not `TwoWay`. Anyway, I'll leave it open for you to double check and close.